### PR TITLE
Rspamd 1.7.0

### PR DESCRIPTION
**Pull request by stephdl**
Upgrade to 1.7.0, following the rspamd migration guide, score are still supported in metrics.conf but we are encouraged to put them in score.d/ therefore my PR

https://rspamd.com/doc/migration.html

BTW: do not know if it is just a feeling but I have the impression the history is quicker now

### CODE REVIEW

File: nethserver-mail2.spec [hunk 1]
```diff
@@ -25,7 +25,7 @@ Summary: Enforces anti-spam and anti-virus checks on any message entering the ma
 BuildArch: noarch
 Requires: nethserver-mail2-common, nethserver-antivirus
 Requires: nethserver-dnsmasq, nethserver-unbound
-Requires: rspamd
+Requires: rspamd >= 1.7.0
```
Comment (DavidePrincipi) [inline nethserver-mail2.spec]: The "=" will not ever match because "1.7.0" is missing the Release number

**Comment by DavidePrincipi** [comment]
Is the new bayes db format backward compatible? Do I need to train the Bayesian filters again? If so, please remember to inform beta testers.

**Comment by stephdl** [comment]
yes the new bayes db format reset to zero what the filter learnt, but I feel it worse the loss...we need to document it....we are in a beta stage, do you think it is a bad ?

**Comment by DavidePrincipi** [comment]
It's not a big issue, just add a "release note" for beta testers, by announcing it on community

**Pull request merged**